js/index.js

```diff
@@ -1 +1,65 @@
 // Your code goes here
+
+const headerColor = document.querySelectorAll('h2');
+
+headerColor.forEach((item) => {
+    item.addEventListener("mouseover", event => {
+        event.target.style.color = "#17A2B8";
+    });
+
+    item.addEventListener("mouseout", event => {
+        event.target.style.color = "black";
+    });
+});
+
+const highlight = document.querySelector('.text-content');
+
+highlight.addEventListener("dblclick", event => {
+    event.target.style.backgroundColor = "yellow"
+    });
+
+const buttonClick = document.querySelectorAll('.btn');
+
+buttonClick.forEach( (item) => {
+    item.addEventListener("click", event => {
+        event.target.style.backgroundColor = "pink";
+    });
+});
+
+const headerDrag = document.querySelector('h1');
+// console.log(headerDrag)
+
+    headerDrag.addEventListener("dragstart", () => {
+        headerDrag.style.color = "red";
+        // console.log(event.target.value);
+    });
+
+    headerDrag.addEventListener("drag", event => {
+        event.target.style.backgroundColor = "black";
+        console.dir(event.target);
+    });
+
+    headerDrag.addEventListener("dragend", event => {
+        event.target.style.backgroundColor = "purple";
+    });
```

Comment:
if you target the element by name, using the dragend event, then I got it to turn black again.  here's the code I used.
![image](https://user-images.githubusercontent.com/30554629/60302395-72188900-98f9-11e9-823a-97ed43bca656.png)